test_commercial_license.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
+
+set -e
+
+git_commit() {
+    git commit "$@"
+    local ret=$?
+    # Git has a maximum resolution of one second, and we need --date-order to produce correct output.
+    sleep 1
+    return $ret
+}
+
+do_check() {
+    echo "TEST: $1..."
+    shift
+
+    if "$(dirname "$0")/check_license_go_code.sh" "$@"; then
+        echo PASSED
+        return 0
+    else
+        echo FAILED
+        return 1
+    fi
+}
+
+do_negative_check() {
+    echo "TEST: $1..."
+    shift
+
+    if "$(dirname "$0")/check_license_go_code.sh" "$@"; then
+        echo FAILED
+        return 1
+    else
+        echo PASSED
+        return 0
+    fi
+}
+
+# Copyright 2019 Northern.tech AS
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http:#www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
```

Comment:
This should be at the top

Reply:
Wow! How did it end up here? This was some weird merge logic gone wrong I think...